pyanaconda/bootloader.py

```diff
@@ -121,6 +121,11 @@ def _is_on_iscsi(device):
     return all(isinstance(disk, blivet.devices.iScsiDiskDevice)
                for disk in device.disks)
 
+def _is_ibft(device):
+    """Tells whether a given device is ibft disk or not."""
```

Comment:
Minor thing - I'd name the function _is_on_ibft because, same as in _is_on_iscsi, we are not actually checking if the device (ie /boot partition) is ibft disk but if it is *on* ibft disk(s).